src/render.js

```diff
@@ -28,14 +28,15 @@ export function renderToDiagram(vnode, co = false) {
     return new Diagram({
         ...diagramNode.props,
         co: co !== !!diagramNode.props.co,
+        align: align,
```

Comment:
In ES6, you don't need to write `align: align,` in an object literal, simply `align,` will suffice.